crates/net/network/src/network.rs

```diff
@@ -130,10 +130,20 @@ impl NetworkHandle {
     }
 
     /// Get the current status of the node.
-    pub async fn get_status(&self) -> Result<Status, oneshot::error::RecvError> {
+    pub async fn get_status(&self) -> Result<NetworkStatus, oneshot::error::RecvError> {
         let (tx, rx) = oneshot::channel();
         let _ = self.manager().send(NetworkHandleMessage::GetStatus(tx));
-        rx.await
+        let status = rx.await?;
+
+        Ok(NetworkStatus {
+            client_version: "Reth".to_string(),
```

Comment:
sorry, I should have made this more clear in the issue, we don't the `Status` as the response from the `NetworkManager` we want `NetworkStatus`, so the `NetworkManager` should populate it directly.

because `NetworkManager` already has access to the `client_version` via `SessionManager.hello_message`